test/affprop.jl

```diff
@@ -46,6 +46,14 @@ using Statistics
         @test R.counts[i] == count(==(i), R.assignments)
     end
 
+    @testset "Ensure works on nonbasic array type" begin
```

Comment:
I think it's slightly more informative:
```suggestion
    @testset "non-Matrix{T} distances support" begin
```
Could you please adjust the other testset names as well?